tests/data_protection/oadp/utils.py

```
@@ -1,15 +1,98 @@
+@contextmanager
+def create_windows_vm_from_dv_template(
+    storage_class: str,
+    namespace: str,
+    dv_name: str,
+    vm_name: str,
+    image_url: str,
+    dv_size: str,
+    template_labels: dict[str, Any],
+    client: DynamicClient,
+    cpu_model: str | None = None,
+    wait_running: bool = True,
```

Ruff (0.15.12)

[warning] Boolean-typed positional argument in function definition

(FBT001)